api/src/main/java/org/openmrs/module/idgen/IdgenModuleActivator.java

```diff
@@ -43,4 +46,18 @@ public void stopped() {
 	public void setDaemonToken(DaemonToken token) {
 		IdgenTask.setDaemonToken(token);
 	}
+	
+	private void initPrefixAttributeType() {
```

Comment:
Let's be more specific then:
* `createLocationPrefixAttributeType` or
* `initLocationPrefixAttributeType`

I guess I'd favour the former.